app/models/subscription.rb

```diff
@@ -6,6 +6,8 @@ class Subscription < ApplicationRecord
 
   include Auditor::Model
 
+  attr_reader :subject
```

Comment:
I don't understand how this works - Subscription objects don't have a `subject` column, and I can't see a `subject` method here either.

Reply:
it is needed so that the test can access subject

Reply:
What is the test accessing? A nil value?

Reply:
Maybe you meant `reference` instead of `subject`?